ADRs/RH/SIG-SRE/ADR-00000 Template.md

```diff
@@ -0,0 +1,69 @@
+# ADR-00000 Template
+
+Authors: 
+
+
+## Status
+
+&lt;_Draft, Proposed, Reviewing, Accepted, Rejected, Superseded_> 
+
+
+## Problem Statement
+
+&lt;_Why does this ADR exist? What issue is it trying to solve?_>
+
+
+## Goals
+
+* &lt;_Clearly list the goals which this work will try to achieve._>
+* &lt;_The goals should be achievable, not visionary statements._>
+
+
+## Non-goals
+
+* &lt;_If any of the goals might be interpreted too broadly, here's where you limit their scope._>
+
+
+## Current Architecture
+
+* &lt;_If something is already solving the problem (even partially), here's where you mention it._>
+
+
+## Proposed Architecture
+
+* &lt;_Explain the full overview of the proposed solution.>
```

Comment:
s/full/high level


IOW just enough design to allow for reviewers to understand the decisions proposed by the ADR